kpatch-build/create-diff-object.c

```diff
@@ -1632,8 +1664,13 @@ static void kpatch_check_func_profiling_calls(struct kpatch_elf *kelf)
 		    (sym->parent && sym->parent->status == CHANGED))
 			continue;
 		if (!sym->twin->has_func_profiling) {
+#ifdef __aarch64__
+			log_normal("function %s doesn't have patchable_function_entry,"
+				   " unable to patch\n", sym->name);
+#else
 			log_normal("function %s has no fentry/mcount call, unable to patch\n",
 				   sym->name);
+#endif
```

Comment:
Another unneeded '#ifdef', instead the existing message can be made arch-independent, e.g. "function %s doesn't have patchable function entry".